libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt

```diff
@@ -302,17 +305,36 @@ class MapboxTripNotification constructor(
         val maneuverImageId = getManeuverResource(legStep)
         if (maneuverImageId != currentManeuverId) {
             currentManeuverId = maneuverImageId
-            collapsedNotificationRemoteViews?.setImageViewResource(
-                R.id.maneuverImage,
-                maneuverImageId
-            )
-            expandedNotificationRemoteViews?.setImageViewResource(
-                R.id.maneuverImage,
-                maneuverImageId
-            )
+            when (Build.VERSION.SDK_INT > Build.VERSION_CODES.M) {
```

Comment:
Logic is duplicating, may be better to move it to one place, wdyt?

Reply:
I believe was duplicated on purpose to include the fix in both the legacy `libandroid-navigation` and `1.0` (`libtrip-notification`). So not a problem as `libandroid-navigation` code will be replaced when releasing `1.0`.